src/main/java/org/damap/base/rest/storage/InternalStorageDO.java

```diff
@@ -10,12 +13,18 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class InternalStorageDO {
   private long id;
-  private String url;
-  private String backupFrequency;
-  private String storageLocation;
-  private String backupLocation;
+
+  @NotBlank(message = "name cannot be blank")
+  @NotNull(message = "name cannot be null") private String url;
+
+  @NotBlank(message = "storage location cannot be blank")
+  @NotNull(message = "storage location cannot be null") private String storageLocation;
+
+  @NotBlank(message = "backup location cannot be blank")
```

Comment:
I think this can be empty

Reply:
For the backup location? Can be empty not